encord_agents/core/vision.py

```diff
@@ -111,5 +111,16 @@ def crop_to_object(image: NDArray[np.uint8], coordinates: CroppableCoordinates)
 
 
 def b64_encode_image(img: NDArray[np.uint8], format: Base64Formats = ".jpg") -> str:
+    """
+    Encode an image to a base64 string.
+
+    Args:
+        img: The image to encode. Expects [RGB] channels
+        format: The format of the image.
+
+    Returns:
+        The base64 encoded image.
+    """
+    img = cv2.cvtColor(img, cv2.COLOR_RGB2BGR)  # type: ignore
```

Comment:
better to just have:
```suggestion
    coerced_color_img = cv2.cvtColor(img, cv2.COLOR_RGB2BGR)
```
then no need for type ignore?